app.rb

```diff
@@ -13,8 +13,13 @@ class Uploader < Sinatra::Base
 
     get '/healthcheck' do
       {
+        service_status: 'OK',
```

Comment:
Thanks. I just have a nasty feeling we'd forget otherwise, and end up with a Lego healthcheck that always says "everything is awesome!"